src/Resources/config/liip_imagine.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <parameters>
+        <parameter key="sonata.media.thumbnail.liip_imagine">Sonata\MediaBundle\Thumbnail\LiipImagineThumbnail</parameter>
+    </parameters>
+    <services>
+        <service id="sonata.media.thumbnail.liip_imagine" class="%sonata.media.thumbnail.liip_imagine%">
+            <argument type="service" id="liip_imagine.cache.manager" on-invalid="null"/>
+            <argument type="service" id="inventis.media.liip_imagine.filter_resolver_registry" on-invalid="null"/>
```

Comment:
"inventis"? is that a liip service?

Reply:
@greg0ire sorry, leftover from porting it to sonata from our project